udata/api/oauth2.py

```diff
@@ -111,7 +112,7 @@ def check_redirect_uri(self, redirect_uri):
     def check_client_secret(self, client_secret):
         return self.secret == client_secret
 
-    def check_token_endpoint_auth_method(self, method):
+    def check_endpoint_auth_method(self, method, _endpoint):
```

Comment:
Those additional endpoints and renaming were needed for the new version?

Reply:
So what happened is when running the tests after the authlib upgrade, I had a bunch of errors. For each of those errors, I tracked down the initial cause, and search the old and new docs, and try to find how the lib was modified.

This is one of those, but an easy one, as it was right there [in the new docs](https://docs.authlib.org/en/v1.3.1/django/2/authorization-server.html#client)